Add SVSE2 tournament page (#9818)
ppy/osu-wiki: wiki/Tournaments/SVSE/2/en.md

2. [Tanger - ! PARTY SIRENS ! (H1Pur) \[Second Stage | ! GROOVY TIME !\]](https://osu.ppy.sh/beatmapsets/2007283#mania/4175187)
3. [ALESDA! - Nikoniko (BlackyDay) \[Third Stage | SiVoSiVo\]](https://osu.ppy.sh/beatmapsets/2007317#mania/4175257)
4. [Akira Complex & PSYQUI - Come to me (Orca-) \[Fourth Stage | a cathartic return;\]](https://osu.ppy.sh/beatmapsets/2007311#mania/4175239)

## Results

### Finals

Saturday, 29 July 2023:

| Player A |  |  | Player B | Match Link |
| --: | :-: | :-: | :-- | :-- |
| **gyoza_goki** ::{ flag=JP }:: | **3** | 0 | ::{ flag=CA }:: walmart5193 | [#1](https://osu.ppy.sh/community/matches/109729917) |
| **ICECAKE** ::{ flag=JP }:: | **3** | 1 | ::{ flag=US }:: ERA Basil | [#1](https://osu.ppy.sh/community/matches/109730191) |

Monday, 31 July 2023:

| Player A |  |  | Player B | Match Link |
| --: | :-: | :-: | :-- | :-- |
| Kamikho ::{ flag=PE }:: | 0 | **3** | ::{ flag=VN }:: **mightyxd** | [#1](https://osu.ppy.sh/community/matches/109770469) |
| ERA Punish ::{ flag=DE }:: | 0 | **3** | ::{ flag=VN }:: **Lott** | [#1](https://osu.ppy.sh/community/matches/109770747) |

### Week 4

> Teams with their name on bold are advance to finals.
> Teams with their name crossed out are eliminated.

Reviewer: ```suggestion
Teams with emboldened names advance to finals.
Teams with names crossed out are eliminated.
```

| Position | Team | Side A Position | Side B Position | Avg. Position | Team Score |
| --: | :-: | :-: | :-: | :-: | --: |
| #1 | **gaki wa oshimai!** | 2 | 3 | 2.5 | 1,989,158 |
| #2 | **mlem** | 1 | 6 | 3.5 | 1,985,314 |
| #3 | **no we're not boyfriends** | 7 | 1 | 4 | 1,983,572 |
| #4 | **The Hard Balls** | 8 | 2 | 5 | 1,977,465 |
| #5 | Descendants of Arccat | 6 | 5 | 5.5 | 1,977,659 |
| #6 | KuruMi KonbanMi | 5 | 7 | 6 | 1,977,037 |
| #7 | Makuron de misshon | 4 | 9 | 6.5 | 1,966,813 |
| #8 | ~~:yangu:~~ | 3 | 10 | 6.5 | 1,963,982 |
| #9 | ~~Billy et son chien~~ | 10 | 4 | 7 | 1,954,437 |
| #10 | ~~kouvola mcdonalds meetup~~ | 9 | 8 | 8.5 | 1,960,057 |
| #11 | ~~broke on mems~~ | 11 | 11 | 11 | 955,101 |

### Week 3

> Teams with their name on italics are on last chance situation.
> Teams with their name crossed out are eliminated.

Reviewer: ```suggestion
Teams with italicised names are on last chance situation.
Teams with names crossed out are eliminated.
```

| Position | Team | Side A Position | Side B Position | Avg. Position | Team Score |
| --: | :-: | :-: | :-: | :-: | --: |
| #1 | KuruMi KonbanMi | 3 | 1 | 2 | 1,993,885 |
| #2 | Makuron de misshon | 5 | 2 | 3.5 | 1,991,461 |
| #3 | mlem | 2 | 6 | 4 | 1,990,409 |
| #4 | Descendants of Arccat | 7 | 3 | 5 | 1,989,324 |
| #5 | The Hard Balls | 6 | 5 | 5.5 | 1,989,517 |
| #6 | no we're not boyfriends | 8 | 4 | 6 | 1,988,114 |
| #7 | gaki wa oshimai! | 4 | 8 | 6 | 1,983,075 |
| #8 | :yangu: | 1 | 15 | 8 | 1,866,632 |
| #9 | Billy et son chien | 12 | 7 | 9.5 | 1,943,655 |
| #10 | *kouvola mcdonalds meetup* | 10 | 10 | 10 | 1,965,878 |
| #11 | ~~Scrambled Eggs~~ | 11 | 12 | 11.5 | 1,925,755 |
| #12 | *broke on mems* | 15 | 9 | 12 | 1,921,038 |
| #13 | ~~svssy baka~~ | 13 | 11 | 12 | 1,908,569 |
| #14 | ~~Jouœurs rincés~~ | 9 | 18 | 13.5 | 1,687,143 |
| #15 | ~~De Grachtengordelse Schavuiten~~ | 17 | 13 | 15 | 1,878,081 |
| #16 | ~~team a~~ | 14 | 16 | 15 | 1,789,481 |
| #17 | ~~Team at~~ | 18 | 14 | 16 | 1,866,262 |
| #18 | ~~Taiko Main forced into SV~~ | 16 | 17 | 16.5 | 1,715,982 |

### Week 2

> Teams with their name on italics are on last chance situation.
> Teams with their name crossed out are eliminated.

Reviewer: ```suggestion
Teams with italicised names are on last chance situation.
Teams with names crossed out are eliminated.
```

| #13 | *svssy baka* | 26 | 6 | 16 | 1,953,060 |
| #14 | Taiko Main forced into SV | 15 | 17 | 16 | 1,941,175 |
| #15 | broke on mems | 23 | 10 | 16.5 | 1,949,580 |
| #16 | *Scrambled Eggs* | 16 | 18 | 17 | 1,933,258 |
| #17 | ~~barudak gulanova~~ | 12 | 22 | 17 | 1,925,581 |
| #18 | *Jouœurs rincés* | 10 | 24 | 17 | 1,912,513 |
| #19 | ~~HiraNewsonFlow (by ChatGPT)~~ | 21 | 16 | 18.5 | 1,930,252 |
| #20 | *De Grachtengordelse Schavuiten* | 19 | 19 | 19 | 1,925,208 |
| #21 | ~~Los Simsimi~~ | 25 | 14 | 19.5 | 1,928,388 |
| #22 | ~~Boykissers~~ | 20 | 20 | 20 | 1,918,724 |
| #23 | ~~25110054~~ | 28 | 15 | 21.5 | 1,917,391 |
| #24 | ~~mrrp mrrp meow~~ | 17 | 27 | 22 | 1,818,256 |
| #25 | ~~TescoMealDeal~~ | 18 | 28 | 23 | 1,730,574 |
| #26 | ~~play rabi-ribi after this~~ | 22 | 26 | 24 | 1,815,739 |
| #27 | ~~sata andagi!~~ | 24 | 25 | 24.5 | 1,884,784 |
| #28 | ~~north sentinel island~~ | 30 | 23 | 26.5 | 1,872,987 |
| #29 | ~~:1217angry:~~ | 27 | 29 | 28 | 1,682,550 |
| #30 | ~~3 tortues trotaient sur 1 toit~~ | 29 | 30 | 29.5 | 1,387,927 |
| #31 | ~~Schizo Velocity~~ | 31 | 32 | 31.5 | 895,544 |
| #32 | ~~cta lovers~~ | 32 | 32 | 32 | 0 |

### Week 1

> Teams with their name on italics are on last chance situation.

Reviewer: ```suggestion
Teams with italicised names are on last chance situation
```

| #15 | team a | 3,857,094 |
| #16 | svssy baka | 3,850,488 |
| #17 | Jouœurs rincés | 3,840,449 |
| #18 | sata andagi! | 3,839,344 |
| #19 | Taiko Main forced into SV | 3,836,334 |
| #20 | mrrp mrrp meow | 3,820,034 |
| #21 | play rabi-ribi after this | 3,817,033 |
| #22 | Scrambled Eggs | 3,805,346 |
| #23 | Boykissers | 3,788,681 |
| #24 | north sentinel island | 3,772,590 |
| #25 | TescoMealDeal | 3,769,282 |
| #26 | De Grachtengordelse Shavuiten | 3,757,118 |
| #27 | :1217angry: | 3,751,494 |
| #28 | broke on mems | 3,717,879 |
| #29 | cta lovers | 3,713,704 |
| #30 | Schizo Velocity | 3,670,547 |
| #31 | HiraNewsonFlow (by ChatGPT) | 3,665,213 |
| #32 | 3 tortues trotaient sur 1 toit | 3,641,099 |
| #33 | Chomper From PVZ | 3,633,886 |
| #34 | gensokyo harem | 3,624,928 |
| #35 | emballé c'est pesé | 3,614,387 |
| #36 | vingt-sept | 3,533,487 |
| #37 | 不玩原神不玩的 | 3,532,877 |
| #38 | Nepka Hunters | 3,504,400 |
| #39 | CrôälaïnË | 3,452,041 |
| #40 | Kim Petras ft. Mili | 3,369,037 |
| #41 | Beans On Toast | 3,303,355 |
| #42 | Forniteloliclub | 3,050,612 |
| #43 | SVn't | 2,705,243 |
| #44 | Losmatapro | 2,529,694 |

## Ruleset

### Overall notes

> 1. SV: Summer Exhibition 2 is a **team tournament**, played on the osu!mania game mode.

Reviewer: ```suggestion
1. SV: Summer Exhibition 2 is an osu!mania **team tournament**.
```

  - Teams are formed with 2 players each.
2. The scoring system used will be **ScoreV2**.
3. There will be a Qualifier Stage, 4 weeks of leaderboard play and a final single elimination stage.
  - **The maximum number of teams who can advance to the weekly stages is 32**.
  - **Top 4 teams by the end of week 4 will advance to the final stage**.
4. Every week an "Information Compass" will be released of each map of the pool which will give players important information on the content of each chart.
  - The Information Compass is a 5x5 grid where the X-Axis represents the chart type bias and the Y-Axis represents the complexity bias.
5. Mappool will be released on different dates depending of the stage:
  - **Qualifiers and Final stage**: Mappool will be released one week in advance, right after the mappool showcase.
  - **Weekly stages**: Mappool will be released every Friday at 0 UTC.
6. Maps will be played with [FreeMod](/wiki/Gameplay/Game_modifier#freemod). The only allowed mods are FadeIn, Flashlight, Hidden, and Mirror.

### Tournament structure

#### Qualifiers

1. The Qualifier pool contains 4 maps, all of which will use [FreeMod](/wiki/Gameplay/Game_modifier#freemod) rules.

> 2. Teams will select one player to play each map and each player will play two different maps from the mappool.

Reviewer: this reads convoluted, any chance you could rephrase/simplify it?

> 3. When joining the lobby they will tell the referee what maps each player will be playing.
>   - Players cant change what maps are they going to play after this.

Reviewer: ```suggestion
3. When joining the lobby, the teams will tell the referee what maps each player will be playing.
  - From this point, players can't change selected maps.
```

4. Each team will have two tries at each map, their best score will count for seeding.
5. The top 32 seeded teams will advance to the weekly stages.
  - This seeding is done by the sum of all the best scores in each map.

#### Weekly stages

1. There will be 4 weeks of match play.
2. Every week 2 maps will be released, one for "Side A" and another for "Side B".
3. An Information Compass will be released for each of these maps every Monday at 0 UTC.
4. After the Information Compasses are released, teams will sign up one player to the Side A lobby, and the other for the Side B lobby.
  - They can do this until Thursday 23:59 UTC.
5. On Friday at 0 UTC the maps are uploaded and posted in the server.
6. Each player will have three tries at each map, their best score will count for each weekly stage.
7. Each map will have a leaderboard with the best recorded scores on it, and their placement on those will be used for the overall week placement
8. Teams advance to the next week stage if they are not eliminated, elimination happens if a team fulfills one of the folowing conditions:

>   - They get 2 "Last Chance" flags, these flags are given by being on the bottom half of the leaderboard each week.

Reviewer: ```suggestion
  - They get 2 "Last Chance" flags, which are given to the teams in the bottom half of the leaderboard each week.
```

  - They are one of the bottom 4 teams on the leaderboard on a given week.
9. Top 4 teams on the leaderboard by week 4 advance to finals.

>   - Teams outside of top 4 that dont get eliminated by the end of week 4 are survivors and get a small prize.

Reviewer: ```suggestion
  - Teams outside of top 4 not eliminated by the end of week 4 are considered survivors and get a small prize.
```

#### Final Stage

1. Teams that advanced to finals will play a small single-elimination bracket.
1. The Finals pool contains 5 maps, all of which will use [FreeMod](/wiki/Gameplay/Game_modifier#freemod) rules.
2. Matches are defined as follows:
  - First place faces Fourth place.
  - Second place faces Third place.
  - Losers of each match will define the 3rd and 4th places of the tournament.
  - Winners of each match will define the Grand Champion of SVSE2.
3. Teams needs to win 2 sets to win a match, to win a set they need to win 3 maps (best of 5).
4. Teams will select an "A player" and a "B player", the A player of each team will face each other, similar to the B player on a 1 versus 1 set. These will be the first and second set of the match.

>   - These A and B players cant be swapped for the entire stage.

Reviewer: ```suggestion
  - These A and B players can't be swapped for the entire stage.
```

5. If both teams tie on set points by the end of the second set, a third set will be played, this time a 2 versus 2.

> 6. Each team must use `!roll` command at the start of every set in `#multiplayer`.

Reviewer: ```suggestion
6. Each team must use the `!roll` command at the start of every set in `#multiplayer`.
```

   - The winner of the `!roll` starts picking first.
   - The loser of the `!roll` picks second.

> 7. There wont be bans in this stage.

Reviewer: ```suggestion
7. There will be no bans in this stage.
```
